src/openqaoa-core/algorithms/qaoa/qaoa_result.py

```diff
@@ -42,20 +30,28 @@ def most_probable_bitstring(cost_hamiltonian, measurement_outcomes):
     }
 
 
-class Result:
+class QAOAResult:
```

Comment:
Why `QAOAResult` and not `QAOAResults`? 
For rqaoa we have `RQAOAResults`, and for workflows we have `q.results`.

Reply:
I think we should probably remove the `s` for all ...

Reply:
yes

Reply:
Resolved in commit [a52535a](https://github.com/entropicalabs/openqaoa/pull/170/commits/a52535a3691816acfd4139b69ba58105dfa01c35)